src/CNGLOB.H

```diff
@@ -84,17 +84,28 @@ typedef long long LLI;		// signed 64 bit integer
 typedef unsigned long long ULLI;
 //typedef int BOOL;	if needed: // 16 or 32 bit Boolean, matches windows.h.
 
-#undef ELIMINATE_WINDOWS		// #define to eliminate Windows dependencies
-								//   work in progress, 10-2017
-#if defined( ELIMINATE_WINDOWS)
-// typedefs based on windows.h
+/*---------------------------- Remove DLL Support ---------------------------*/
+#undef SUPPORT_DLLS
+
+#if 1
+#define	ELIMINATE_WINDOWS
+#else
+#undef	ELIMINATE_WINDOWS		// #define to eliminate Windows dependencies
+#endif 								//   work in progress, 10-2017
```

Comment:
You don't need the if-statement here. Instead, you can have one line with `#define ELIMINATE_WINDOWS` that you can comment out if you don't want it defined.